Automattic-Tracks-iOS/Services/EventLoggingUploadQueue.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+open class EventLoggingUploadQueue {
```

Comment:
Similar to my previous comment, does this need to be an `open class`?

Also, this class doesn't seem to be used, I am guessing you're planning to use it in a different PR.

Reply:
Same justification about it being `open`. 

Yep – the next PR will handle putting things on the queue, then taking them off uploading them.